src/clean.c

```diff
@@ -30,16 +30,15 @@ static void print_help()
 {
 	fprintf(stderr,
 		"Usage:\n"
-		"	swupd clean [options]\n"
+		"	swupd clean [OPTION...]\n\n"
```

Comment:
the usage message for swupd check-update is using spaces to indent, in this case you are using a tab, it would be nice to be consistent in indentation across all the usage messages.

Reply:
true